### DISCUSSION TITLE
We need widget icon one click to start recorder/stop

### DISCUSSION THREAD

**Discussion by anwikipedia** [anonfaded/FadCam#31] [Ideas]
We need widget icon one click to recorder .
This is first time i install this app and i can't find the widget button one click recorder/stop.
Did i have to turn on something so the widget work?

**Comment by anonfaded** [comment]
> We need widget icon one click to recorder .
> This is first time i install this app and i can't find the widget button one click recorder/stop.
> Did i have to turn on something so the widget work?

This feature isn't implemented yet but will soon be added.

Someone made issue for this too: https://github.com/anonfaded/FadCam/issues/10#issue-2407543570

**Reply by anonfaded** [reply]
@anwikipedia This feature has been added. Check the beta version tag please.

**Comment by monkeyotg** [comment]
@anonfaded Thanks for your work!
One question, would you consider adding real widgets, I mean not just some shortcuts by long pressing?

**Reply by anonfaded** [reply]
Those shortcuts are actually widgets. Like when you long press on app icon you see the shortcuts, and then again long press on the shortcut and drag & drop to your homescreen. Then just with single tap it will trigger recording without opening the app. I guess you didn't drag and drop the shortcut icons to home screen.
And thank you:)

**Reply by monkeyotg** [reply]
> Those shortcuts are actually widgets. Like when you long press on app icon you see the shortcuts, and then again long press on the shortcut and drag & drop to your homescreen. Then just with single tap it will trigger recording without opening the app. I guess you didn't drag and drop the shortcut icons to home screen. And thank you:)

Thanks for the explanation!
I haven't used a standard launcher or anything similar for a long time. All the launchers I've used in recent years don't handle with widgets in that way, so missed that point.

Well I will try some other ways.

Thanks again~

**Reply by anonfaded** [reply]
yeah btw i'm planning to have some better widgets as well. But these shortcuts currently function similarly as well as it doesn't open the app on tap. And if you want to use keymapper apps to use your android volume buttons for starting or stopping recording; these shortcuts works with that as well

**Reply by monkeyotg** [reply]
> yeah btw i'm planning to have some better widgets as well. But these shortcuts currently function similarly as well as it doesn't open the app on tap. And if you want to use keymapper apps to use your android volume buttons for starting or stopping recording; these shortcuts works with that as well

Thanks, that's fine, not very urgent. I'm using a minimalist launcher now. Because I don't want a messy home screen anymore. It supports gesture-widget binding.
